alts/src/main/java/io/grpc/alts/AltsAuthContext.java

```diff
@@ -14,14 +14,21 @@
  * limitations under the License.
  */
 
-package io.grpc.alts.internal;
+package io.grpc.alts;
 
 import com.google.common.annotations.VisibleForTesting;
+import com.google.protobuf.Any;
+import io.grpc.InternalChannelz.OtherSecurity;
+import io.grpc.InternalChannelz.Security;
+import io.grpc.alts.internal.AltsContext;
+import io.grpc.alts.internal.HandshakerResult;
+import io.grpc.alts.internal.RpcProtocolVersions;
```

Comment:
We can't be returning internal classes, unless the method returning it is internal as well. But if it is internal, nobody other than ALTS itself should call it. SecurityLevel thus seems inappropriate for internal. Unclear about RpcProtocolVersions and HandshakerResult.

Reply:
I think authn/z frameworks need to incorporate the security level into their decisions. I think the same is true for RpcProtocolVersions as soon as there are multiple (at least now MAX_RPC_VERSION_ constants seems to be equal to MIN_RPC_VERSION_ constants). This way one could for example only allow peers with a certain security level and rpc version to access certain methods. Therefore, the according proto's java package shouldn't be internal.  WDYT? Shall I change the proto java_package and dependencies in a new PR to not make this review more complicated?

HandshakerResult is not returned, but only used as constructor argument. I guess this on purpose to not allow arbitrary code to construct it (tests can use getDefaultInstance method).